src/pages/accounts/SignInPage.tsx

```diff
@@ -40,24 +41,23 @@ const TextLinkContainer = styled.div`
 `;
 
 const TextLink = styled(TextButton)`
-	font-size: 0.8rem;
+	font-size: 1rem;
 	font-weight: 500;
 	margin: 0.5em;
+	word-wrap: break-word;
 `;
 
 function validEmail(email: string) {
 	return !!email.match(/.+@.+\..{2,}/);
 }
 
-function tempOnClick() {
-	console.log('A click!');
-}
-
 function SignInPage() {
 	const [email, setEmail] = useState('');
 	const [emailErr, setEmailErr] = useState(false);
 	const [userPass, setUserPass] = useState('');
 
+	const history = useHistory();
```

Comment:
What does the `useHistory()` function do from react-router-dom?

Reply:
So you're definitely familiar with the browser history (the back and forward button). This essentially allows you to access this history through the react-router lib. When you push onto the history, it changes your page and updates the URL. It also allows the user to hit the browser back button to go to the last page they were on, giving the application a multi page feel, while actually being an SPA